This patch was reported upstream already, but it looks like upstream is not
interested in feedback. I've pinged them another time, let's see how it goes.

http://lists.net.ru/pipermail/xneur/2008-November/000909.html

=== modified file 'configure.in'
--- configure.in	2009-03-15 10:29:29 +0000
+++ configure.in	2009-03-15 10:29:44 +0000
@@ -27,8 +27,8 @@
 AM_GNU_GETTEXT_VERSION([0.16.1])
 AM_GNU_GETTEXT([external])
 
-CFLAGS="-Wall -Wextra -Werror -g0 -fPIC -std=gnu99"
-AC_SUBST(CFLAGS)
+DEFAULT_CFLAGS="-Wall -Wextra -Werror -g0 -fPIC -std=gnu99"
+AC_SUBST(DEFAULT_CFLAGS)
 
 AC_OUTPUT([Makefile src/Makefile pixmaps/Makefile glade/Makefile po/Makefile.in])
 

=== modified file 'src/Makefile.am'
--- src/Makefile.am	2009-03-15 10:29:29 +0000
+++ src/Makefile.am	2009-03-15 10:31:43 +0000
@@ -21,5 +21,5 @@
 	xkb.c			\
 	xkb.h
 
-gxneur_CFLAGS = @GTK_CFLAGS@ @XNEUR_CFLAGS@ @GLADE_CFLAGS@
+gxneur_CFLAGS = @GTK_CFLAGS@ @XNEUR_CFLAGS@ @GLADE_CFLAGS@ @DEFAULT_CFLAGS@
 gxneur_LDFLAGS = @GTK_LIBS@ @XNEUR_LIBS@ @GLADE_LIBS@

